helm/tekton-dashboard-loki-proxy/values.yaml

```diff
@@ -15,3 +15,20 @@ replicas: 1
 lokiEndpoint: ""
 orgID: "1"
 since: "120h"
+
+podSecurityContext:
+  runAsGroup: 1000
+  runAsNonRoot: true
+  runAsUser: 1000
+  seccompProfile:
+    type: RuntimeDefault
+securityContext:
```

Comment:
it is a bit confusing why is there a pod wide security context and a loki container only context.

Reply:
Might be just enough to include a comment here saying this is for the loki-proxy pod.